src/components/Table/Table.tsx

```diff
@@ -17,18 +21,39 @@ export const Table = ({
   children,
   fullWidth,
   fixed,
+  scrollable,
+  striped,
+  compact,
+  stackedStyle = 'none',
 }: TableProps): React.ReactElement => {
   const classes = classnames('usa-table', {
     'usa-table--borderless': !bordered,
     [`${styles.fullwidth}`]: fullWidth,
     [`${styles.fixed}`]: fixed,
+    'usa-table--striped': striped,
+    'usa-table--compact': compact && stackedStyle === 'none',
+    'usa-table--stacked': stackedStyle === 'default',
+    'usa-table--stacked-header': stackedStyle === 'headers',
   })
 
   return (
-    <table className={classes} data-testid="table">
-      {caption && <caption>{caption}</caption>}
-      {children}
-    </table>
+    <>
+      {scrollable ? (
+        <div
+          data-testid="scrollable-table"
```

Comment:
Super minor, but since this isn't the actual table, its a container div, maybe it should be called something like `scrollable-table-container`